tests/app/v2/notifications/test_post_notifications.py

```diff
@@ -1895,48 +1896,6 @@ def __send_sms(number_to_send=1):
         response = __send_sms(1)  # attempt to send over limit (11 with max 10)1
         assert response.status_code == 400  # Ensure send is blocked - not sure why we send a 400 here and a 429 everywhere else
 
-    def test_API_BULK_cant_hop_over_limit_2_fragment(self, notify_api, client, notify_db, notify_db_session, mocker):
```

Comment:
There are several other tests involving fragment hopping. Considering the [move away from / removal of sms parts](https://gcdigital.slack.com/archives/CV38DBNVA/p1698960023450489) for counting do we need to keep these tests?

Reply:
Yeah agreed we won't need these hop over tests anymore since it won't work the same way.  For the tests that hop over by 1 though, are they still relevant? Or maybe that functionality is covered elsewhere?

Reply:
I considered keeping the hop by 1 tests. I chose not to because it seemed like we were testing the scenario where a service had 1 fragment left and tried to send a message that had 2 fragments in it. Since it is a 1:1 count now, we won't run into that situation anymore. I think [test_post_sms_not_enough_messages_left](https://github.com/cds-snc/notification-api/blob/f6747d29b52c42a0f90b49ec7a3caf2623dc16fe/tests/app/v2/notifications/test_post_notifications.py#L1474) is adequate.

This does raise an area that I did not test thoroughly though - Where a service has 1 message left and they want to send a job with 2 or more. I will test this more and add tests if necessary but we should be covered.

Reply:
I did some boundary tests trying to go over the SMS limit with jobs. I wasn't able to break anything locally.